src/libAtomVM/module.c

```diff
@@ -196,6 +196,8 @@ Module *module_new_from_iff_binary(GlobalContext *global, const void *iff_binary
     mod->export_table = beam_file + offsets[EXPT];
     mod->atom_table = beam_file + offsets[AT8U];
     mod->fun_table = beam_file + offsets[FUNT];
+    mod->str_table = beam_file + offsets[STRT] + 8;
```

Comment:
why + 8 here?

Reply:
I think you're right, for consistency, I should remove it.

The strings table, as I understand it, is a sequence of strings, all catenated together into essentially one long binary.  The assembler will write the offset in that binary of the start of a given string literal in the beam file, and that offset will show up in the opcode arguments.  I was trying to make it easier to get to the start of the strings table, when looking up a string, but I think it might be more clear if we write a function in `module.c` to return a point to the start of the string at a given offset.

Reply:
Pushed changes.

Reply:
thanks